src/clj/fluree/db/query/sparql/translator.cljc

```diff
@@ -115,6 +116,7 @@
   (let [f (get supported-scalar-functions func)]
     (case f
       "abs"    (str "(" f " " (str/join " " (mapv (comp literal-quote parse-term) args)) ")")
+      "bnode"  (or (str "(" f " " (parse-term (first args)) ")") (str "(" f " " (literal-quote (parse-term (first args))) ")") nil)
```

Comment:
You don't need the `or` in this statement, it will never evaluate the second two arguments. It can just be
```
(str "(" f " " (literal-quote (parse-term (first args))) ")")
```
`literal-quote` will do the right thing regardless of its input, though I expect 99% of the time the argument will be a variable.